snippets/man_gen/hubl_custom_module_fields.json

```diff
@@ -486,5 +486,111 @@
       "}"
     ],
     "description": "HubSpot Font Field"
+  },
+  "Style Tab Group": {
+    "prefix": "styleTab.group",
+    "body": [
+      "{",
+      "\"name\": \"${1:style}\",",
+      "\"label\": \"${2:Style}\",",
```

Comment:
Nit thing but would it make sense to swap label and name to be Styles so that it matches the label on the current tab?